tests/unittests/CoreGraphics/CGBitmapContextTests.mm

```diff
@@ -44,24 +44,36 @@
     CGContextRelease(context);
 }
 
-DISABLED_TEST(CGBitmapContext, BitmapInfoAPIs_RGB) {
+TEST(CGBitmapContext, BitmapInfoAPIs_RGB) {
```

Comment:
Sadly not all of it will work against the reference platform, we already have a caveat regarding that.

---
In reply to: [87862080](https://github.com/Microsoft/WinObjC/pull/1371#discussion_r87862080) [](ancestors = 87862080)